dagql/call/id.go

```diff
@@ -300,24 +279,18 @@ func (id *ID) Append(
 // given metadata changed.
 // customDigest, if not empty string, will become the ID's digest.
 // tainted sets the ID's tainted flag.
-func (id *ID) WithMetadata(customDigest digest.Digest, tainted bool) *ID {
+func (id *ID) WithMetadata(customDigest digest.Digest) *ID {
```

Comment:
suggestion: we can just rename this to `WithDigest` now (and remove `tainted` from the doc string)